### DISCUSSION TITLE
ATOM: Compiling takes for ever after the last update two updates (last working version 11.0.0.1)

### DISCUSSION THREAD

**Discussion by balou1974** [arendst/Tasmota#14984] [Q&A] [answered]
### PROBLEM DESCRIPTION

Compiling takes for ever after the last update two updates (last working version 11.0.0.1)

[env:tasmota-LCD-DE]
build_flags             = ${env.build_flags} -DUSE_DISPLAY_LCD -DUSE_I2C -DMY_LANGUAGE=de_DE

Compiling .pio\build\tasmota-LCD-DE\lib3c6\IRremoteESP8266\ir_Amcor.cpp.o

... compile time takes for ever ...

I am using Windows 11 with atom and platformio to compile / already cleared caches / deleted .platformio but nothing seems to help.

### REQUESTED INFORMATION
_Make sure your have performed every step and checked the applicable boxes before submitting your issue. Thank you!_

- [ ] Read the [Contributing Guide and Policy](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md) and [the Code of Conduct](https://github.com/arendst/Tasmota/blob/development/CODE_OF_CONDUCT.md)
- [x] Searched the problem in [issues](https://github.com/arendst/Tasmota/issues)
- [x] Searched the problem in [discussions](https://github.com/arendst/Tasmota/discussions)
- [x] Searched the problem in the [docs](https://tasmota.github.io/docs/FAQ)
- [ ] Searched the problem in the [chat](https://discord.gg/Ks2Kzd4)
- [wemos d1 mini clone] Device used (e.g., Sonoff Basic): _____
- [ ] Tasmota binary firmware version number used: _____
  - [ ] Pre-compiled
  - [x] Self-compiled
- [ ] Flashing tools used: _____
- [ ] Provide the output of command: `Backlog Template; Module; GPIO 255`:
```lua
  Configuration output here:


```
- [ ] If using rules, provide the output of this command: `Backlog Rule1; Rule2; Rule3`:
```lua
  Rules output here:


```
- [ ] Provide the output of this command: `Status 0`:
```lua
  STATUS 0 output here:


```
- [ ] Set `weblog` to 4 and then, when you experience your issue, provide the output of the Console log:
```lua
  Console output here:


```

### TO REPRODUCE
_Steps to reproduce the behavior:_

### EXPECTED BEHAVIOUR
_A clear and concise description of what you expected to happen._

### SCREENSHOTS
_If applicable, add screenshots to help explain your problem._

### ADDITIONAL CONTEXT
_Add any other context about the problem here._

**(Please, remember to close the issue when the problem has been addressed)**

**Comment by Jason2866** [comment, marked as answer]
There is no general problem. Every Tasmota commit is checked via CI if it compiles successfully and for every commit the firmware is available here https://github.com/arendst/Tasmota-firmware
Have you tried to compile a standard firmware (already defined)? If this works, there is something wrong with your custom defined env

**Comment by balou1974** [comment]
Yesterday I tried same setting with gitpod and it worked - I think something messed up my buildsystem (maybe the new arduino core)? The only things I change are the ini-files to get german as language and some special things like i2c, lcd-display, zigbee ... I wonder what happend - maybe I should switch to the linux buildsystem.

**Comment by Jason2866** [comment]
I used ATOM a looong time ago. Switched to VSC never regret this change.
